### ISSUE TITLE
1.5.0 update broke Octolapse and webcam performance

### ISSUE THREAD

**Issue by badpixelvfx** [OctoPrint/OctoPrint#3842]
#### Did the same happen when running OctoPrint in safe mode?

Update broke OctoLapse plugin.  Following update, interface was garbled and fields were missing.  "Octolapse Loading" text and spinning icon permanently displaying at bottom of plugin panel.

Logitech webcam also behaving unexpectedly.  Prior to update, imaging was fine, afterwards, preview was locked to 4:3 with black bars filling the rest of the 16:9 space when 16:9 was the selected ratio.  When 4:3 was selected, ratio changed and filled preview window without black bars.

#### Version of OctoPrint

1.5.0

#### Operating System running OctoPrint

OctoPi 0.17.0

#### Printer model & used firmware incl. version

Mk3s

#### Browser and version of browser, operating system running browser

Chrome, Windows 17763.652

#### Link to octoprint.log

Did not collect.  Reverted to 1.4.2 and everything started working again.

#### Screenshot(s)/video(s) showing the problem:

https://drive.google.com/file/d/1MtvJVuWNvqIzhvq19utnrxxM0ZI6KaSM/view?usp=sharing

https://drive.google.com/file/d/1tne3elQ-C59p7fwZmPhD9rZ_00ppL8Zv/view?usp=sharing

I have read the FAQ.

**Comment by jneilliii** [comment]
there is an updated version to OctoLapse that fixes issue relative to OctoPrint 1.5.0. This is not an issue with OctoPrint but needs to be reported to the plugin's repo, not here. For your convenience, copy/paste the URL below in Plugin Manager > Get More > ...from URL and clicking install if you want to run the updated version.

```
https://github.com/FormerLurker/Octolapse/archive/v0.4.1rc1.zip
```

**Comment by cp2004** [comment]
> Link to octoprint.log
> Did not collect

Since this is a 3rd party issue and has been solved by the author, this can be closed.
